widget/IncompatiblePointDistribution.tsx

```diff
@@ -65,10 +65,22 @@ class IncompatiblePointDistribution extends Component<IncompatiblePointDistribut
 
     if (this.props.selectedDataPoint != null) {
       // Sort the data into the dataRows based on the ordering of the sorted classes
+      var totalErrors = 0;
+      for (var i=0; i < this.props.selectedDataPoint.sorted_classes.length; i++) {
+        var instanceClass = this.props.selectedDataPoint.sorted_classes[i];
+        var dataRow = this.props.selectedDataPoint.h2_error_instance_ids_by_class.filter(
+          dataDict => (dataDict["class"] == instanceClass)).pop();
+        totalErrors += dataRow["errorInstanceIds"].length;
+      }
+
+      if (totalErrors == 0) {
+        totalErrors = 1;
```

Comment:
Might be helpful to have a comment that this is to prevent division by 0 later on.

Reply:
Thank you. I will add that.